src/acrcssc/azext_acrcssc/helper/_taskoperations.py

```diff
@@ -99,10 +103,37 @@ def _create_cssc_workflow(cmd, registry, schedule_cron_expression, resource_grou
         dry_run
     )
 
-    logger.warning(f"Deployment of {CONTINUOUS_PATCHING_WORKFLOW_NAME} tasks completed successfully.")
+    if not silent_execution:
+        print(f"Deployment of {CONTINUOUS_PATCHING_WORKFLOW_NAME} tasks completed successfully.")
+
 
+def _update_cssc_workflow(cmd, registry, schedule_cron_expression, resource_group, dry_run, task_list):
+    # compare the task definition to the existing tasks, if there is a difference, we need to update the tasks
+    # if we need to update the tasks, we will update the cron expression from it
+    # if not we just update the cron expression from the given parameter
+    for task in task_list:
```

Comment:
Do we still need this check as we get the task_list from task existence check?

Reply:
since the caller, retrieves the list of tasks we have it is not needed, unless the function is reused later. Will remove the double check for now.